hudi-flink-datasource/hudi-flink/src/main/java/org/apache/hudi/configuration/FlinkOptions.java

```diff
@@ -799,6 +799,12 @@ private FlinkOptions() {
       .defaultValue(false)
       .withDescription("Skip the _ro suffix for Read optimized table when registering, default false");
 
+  public static final ConfigOption<Boolean> HIVE_SYNC_SKIP_RT_SUFFIX = ConfigOptions
+          .key("hive_sync.skip_rt_suffix")
+          .booleanType()
+          .defaultValue(false)
```

Comment:
may you add sinceValue("0.12.1") to track in the doc when to use?

Reply:
Yes, I'd love to, but there's no since method. Any other suggestions?